import_export_extensions/static/import_export_extensions/css/admin/import_result_diff.css

```diff
@@ -4,3 +4,119 @@ ins {
 del {
   color: black;
 }
+
+.import-preview .errors {
```

Comment:
I see this code from original `django-import-export` package, can reuse it instead of copying?

Recently we decided to include `django-import-export` to dependencies of this package, so I suggest just include "import_export/import.css" file in `celery_import_results.html`